src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -82,6 +84,37 @@ public void run() {
 		}
 	}
 
+	private void checkForWarningAndApplyChanges( final ReviewResult reviewResult ) throws IOException {
+		final List<Boolean> shouldApplyChanges = new ArrayList<>( Arrays.asList( false ) );
+		for ( final AttributeDifference attributeDifference : reviewResult.getAllAttributeDifferences() ) {
+			if ( attributeDifference.getElementIdentificationWarning() != null ) {
+				shouldApplyChanges.add( true );
+			}
+		}
+		if ( shouldApplyChanges.contains( true ) ) {
+			WarningUtil.logWarnings( reviewResult );
+			logger.info( "Are you sure you want to continue? [Yes/No] or [Y/N]" );
+			inputDataInformation( reviewResult );
+		} else {
+			applyChanges( createSutStatePersistence(), reviewResult );
+		}
+	}
+
+	private void inputDataInformation( final ReviewResult reviewResult ) throws IOException {
+		try ( final ConsoleReader reader = new ConsoleReader() ) {
+			final String input = reader.readLine();
+			final String inputValue = input.toLowerCase();
```

Comment:
I would name them as what they are:

```java
final String input = reader.readLine();
final String lowerCaseInput = input.toLowerCase();
```

BTW: I don't know JLine, so can `readLine()` be `null`? Are we supposed to close `ConsoleReader` (as it is `System.in`)?

Reply:
Yes, readline() can be null, but the ConsoleReader constructor can not be only with the parameter "System.in". It can be default or for example:
```suggestion
			final ConsoleReader reader = new ConsoleReader(System.in, System.out);
```